Formula/cjose.rb

```diff
@@ -0,0 +1,32 @@
+class Cjose < Formula
+  desc "C library implementing Javascript Object Signing and Encryption (JOSE)"
```

Comment:
The description for the `jose` formula is

```
  desc "C-language implementation of Javascript Object Signing and Encryption"
```

Is there anything that distinguishes the cjose implementation?

Reply:
I think these are just different implementations, this one by Cisco.